src/ansys/fluent/core/solver/error_message.py

```diff
@@ -21,6 +21,18 @@ def allowed_name_error_message(
     return message
 
 
+def allowed_values_error_message(
+    context: str, trial_name: str, allowed_values: Any
```

Comment:
Should the type-hint for allowed values be something like `Collection[str]`?

Reply:
Actually I added it for following scenario where allowed values are not of `str` type. e.g range, enum

![image](https://github.com/ansys/pyfluent/assets/106588300/aa6041a5-5c16-4335-91c1-c10ca3af00d7)

Reply:
But now just updated type-hint in `allowed_name_error_message` from `allowed_values: List[str]` to `allowed_values: Any`

Reply:
Done. Thank you.